scripts/install/backend.sh

```diff
@@ -12,3 +12,20 @@ if [[ $target == *"prod"* ]]; then
 else
     pip install -e .[dev]
 fi
+
+
+
+read -r -p "Would you like to install required NLTK word lists for machine learning workers? [Y/n] " response
+case "$response" in
+  [yY][eE][sS]|[yY])
+    echo "Installing..."
+    python -m nltk.downloader stopwords > logs/stopwords.log
+    python -m nltk.downloader punkt >logs/punkt.log
+    python -m nltk.downloader popular >logs/popular.log
+    python -m nltk.downloader universal_tagset >logs/universal_tagset.log
+    echo "Done!"
+    ;;
+  *)
+    echo "Skipping frontend dependencies..."
+    ;;
+esac
```

Comment:
There are several issues with this script that need to be addressed.
1. This section of code **CANNOT** go in `backend.sh`, for several reasons. First, it has nothing to do with Augur's backend; it's related to the workers. Second, this script is run as part of several automated builds (like the Docker build, the Travis build, and the `make rebuild` command) that do not have access to `stdin`, so reading user input is impossible and will cause those builds to fail. I recommend moving this code block to its own file, and running it **DURING THE REGULAR INSTALL PROCESS ONLY,** or when the stopwords are required.
2. The `logs/` directory that these words are written to gets completely cleared every time Augur starts, meaning that the words will be deleted before they are ever even used. They need to go in a different location, preferably somewhere close to where they are used in the relevant workers.
3. The output of those commands should not go into `.log` files. The `make clean` process will see those files and delete them; they need to be in `.txt` or some other form.
4. Line 29 is incorrect - these are not frontend dependencies. I assume this is left over from a c/p, but it sill needs to be fixed.
5. This section should output where it stored the words, for posterity's sake.

Reply:
@ccarterlandis : I figured it was wrong, but was not certain why. Thanks for the clear explanation. 
I am working on a solution that addresses the installed base, who will only do `make rebuild-dev`, and do not want to introduce anything that makes installation or maintenance more complex. Nor do I want to install word like these without giving the user a choice. By default, they are placed in a user's home directory. I could specify a directory in `/usr/local/`, which sounds like it would be your preference. Its a worker's issue for sure. 

These CANNOT be installed using Python, at least according to my internet searches. I will find another way and see if it meets the salient criteria.

Reply:
I have a fix. I am testing it now.

Reply:
@ccarterlandis : OK, I fixed this, and specified a directory as you suggested. 
On install: The user is given a choice. 
On rebuild-dev: The user is given a message informing them what is being done. 

In both cases I have created a new script under `scripts/install` to address the installation consistently. 

I'm not sure its perfect, so I look forward to your next review!

Reply:
I have tested the solution twice locally.